x-pack/filebeat/docs/inputs/input-gcs.asciidoc

```diff
@@ -402,12 +402,14 @@ The GCS APIs don't provide a direct way to filter files based on the timestamp,
 [float]
 ==== `retry`
 
-This attribute can be used to configure a list of sub attributes that directly control how the input should behave when a download for a file/object fails or gets interrupted. The list of sub attributes are as follows :-
+This attribute can be used to configure a list of sub attributes that directly control how the input should behave when a download for a file/object fails or gets interrupted.
 
-    1. `max_attempts`: This attribute defines the maximum number of retry attempts(including the initial api call) that should be attempted for a retryable error. The default value for this is `3`. 
-    2. `initial_backoff_duration`: This attribute defines the initial backoff time in seconds. The default value for this is `1s`.
-    3. `max_backoff_duration`: This attribute defines the maximum backoff time in seconds. The default value for this is `30s`.
-    4. `backoff_multiplier`: This attribute defines the backoff multiplication factor. The default value for this is `2`.
+    - `max_attempts`: This attribute defines the maximum number of retry attempts(including the initial api call) that should be attempted for a retryable error. The default value for this is `3`. 
+    - `initial_backoff_duration`: This attribute defines the initial backoff time. The default value for this is `1s`.
+    - `max_backoff_duration`: This attribute defines the maximum backoff time. The default value for this is `30s`.
+    - `backoff_multiplier`: This attribute defines the backoff multiplication factor. The default value for this is `2`.
+
+NOTE: Attributes such as `initial_backoff_duration` and `max_backoff_duration` can be defined in the following formats: `{{x}}s`, `{{x}}m`, `{{x}}h`, here `s = seconds`, `m = minutes` and `h = hours`. The value `{{x}}` can be anything we wish.
```

Comment:
I think something simpler like "NOTE: The `initial_backoff_duration` and `max_backoff_duration` attributes must have time units. Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".

Reply:
Updated